radio/src/targets/simu/opentxsimulator.cpp

```diff
@@ -434,22 +436,14 @@ void OpenTxSimulator::rotaryEncoderEvent(int steps)
     last_tick = now;
   }
 #else
-  // TODO : this should probably be handled in the GUI
   int key;
-#if defined(PCBXLITE)
-  if (steps > 0)
-    key = KEY_DOWN;
-  else if (steps < 0)
-    key = KEY_UP;
-#elif defined(KEYS_GPIO_REG_PLUS) && defined(KEYS_GPIO_REG_MINUS)
-  if (steps > 0)
-    key = KEY_MINUS;
-  else if (steps < 0)
-    key = KEY_PLUS;
-  else
-#endif
-    // Should not happen but Clang complains that key is unset otherwise
-    return;
+  if (keyIsSupported(KEY_UP) | keyIsSupported(KEY_DOWN)) {
+    key = steps > 0 ? KEY_DOWN : KEY_UP;
+  } else if (keyIsSupported(KEY_PLUS) | keyIsSupported(KEY_MINUS)) {
+    key = steps > 0 ? KEY_MINUS : KEY_PLUS;
+  } else {
+    return; // not supposed to happen???
```

Comment:
Is this a "not supposed to happen" that is important enough to know that it should fail the build, or will  something else kack the build first?

Reply:
> "not supposed to happen"

In the sense that we support radios that have either PLUS/MINUS keys, or UP/DOWN, or a rotary encoder. For those that support none of these, there is no useful things that can be done with the mouse wheel.